calls/2024/2024-03-14_DFDL-WG-Call.md

```diff
@@ -0,0 +1,167 @@
+# OGF DFDL Working Group Call Agenda & Minutes 14 March 2024
+
+### Meeting Time
+11:00 - 12:00 EDT.US (UTC-4 so 15:00 - 16:00 UTC)
+
+### Attendees
+* Mike Beckerle
+* Steve Hanson
+* Bradd Kadlecik
+<!--
+* Liam O'Neill
+-->
+
+### IPR statement
+"I acknowledge that participation in this meeting is subject to the OGF Intellectual Property Policy"
+
+### Meeting closed
+11:58 EDT.US
+
+### Next regular call (Monthly on 2nd Thursday)
+2024-04-11 @ 11:00 EDT.US(UTC-4) (15:00 UTC)
+
+## Agenda & Minutes
+
+1. ISO JTC1 PAS update and next steps (action 325)
+
+Besides the problem with the footers being cut off, 
+Steve H noted the 
+- ISO PDF links do not work as well as in the OGF PDF,
+- The page margins are inconsistent (left pages have narrow left margin vs. right pages - 
+  asymmetric), 
+- some bad table splits (& we need to check for more)
+
+Mike will raise these to ISO (Lisa) this week.
+
```

Comment:
We decided to open an action on Dynamic value for dfdl:extraEscapedCharacters property.
That will be 339.